httpclient/src/test/java/feign/httpclient/ApacheHttpClientTest.java

```diff
@@ -188,6 +188,20 @@ public void postWithSpacesInPath() throws IOException, InterruptedException {
                 .hasBody("foo");
     }
 
+    @Test
```

Comment:
Hmm... it's quite consistent to have a test here. There are already quite a few header tests in this class. I've just added one more.